doozer/doozerlib/cli/scan_sources.py

```diff
@@ -412,8 +571,8 @@ def config_scan_source_changes(runtime: Runtime, ci_kubeconfig, as_yaml):
     It will report RHCOS updates available per imagestream.
     """
 
-    runtime.initialize(mode='both', clone_distgits=False, clone_source=False, prevent_cloning=True)
-    ConfigScanSources(runtime, ci_kubeconfig, as_yaml).run()
+    runtime.initialize(mode='both', clone_distgits=False, clone_source=False, prevent_cloning=False)
```

Comment:
Previously `prevent_cloning` was used to ensure we don't clone upstream or distgit repos in scan-sources. With this PR, this job will clone upstream repos when there is a need to rebase into the private repo.
I think we need to be cautious when changing this to False. More testing is needed to ensure we don't clone upstream repos unless there is a need, and ensure that we never clone distgit repos.

Reply:
`prevent_cloning` needs to be set to False, as I verified we'd fall in this codepath:
```
    def clone(self, distgits_root_dir, distgit_branch):
        if self.metadata.prevent_cloning:
            raise IOError('...')
```
I also verified that upstream repos are only cloned when needed during rebase to priv, and that distgits are never cloned.